lib/deep_nest.rb

```diff
@@ -1,10 +1,13 @@
 # frozen_string_literal: true
 
-require_relative "deep_nest/version"
+require_relative 'deep_nest/version'
+require_relative 'ruby_classes'
```

Comment:
We don't want to require this - it will automatically add this to Hashes and Arrays, which we're trying to avoid. But we DO want to `require_relative 'deep_nest/helpers'` in order to make that mix-in available. Otherwise, users will get errors and need to require that file manually themselves.